gslib/tests/test_wildcard_iterator.py

```diff
@@ -499,3 +500,14 @@ def testExistingDirNoFileMatch(self):
         self._test_wildcard_iterator(uri).IterAll(
             expand_top_level_buckets=True))
     self.assertEqual(0, len(res))
+
+  def testExcludeDir(self):
+    """Tests that the exclude regex will omit a nested directory."""
+    exp_uri_strs = self.root_files_uri_strs
+    uri = self._test_storage_uri(suri(self.test_dir, '*'))
+    exclude_pattern = re.compile('file://{}/dir1'.format(self.test_dir))
```

Comment:
Make sure to check Kokoro for Windows given that forward slash is involved. 

Nit: `suri(self.test_dir, 'dir1')` should work as well if you want to follow the `suri` pattern everywhere else.

Reply:
Done.